models/l3ext_cons_lbl.go

```diff
@@ -0,0 +1,109 @@
+package models
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/container"
+)
+
+const (
+	DnL3extConsLbl        = "uni/tn-%s/out-%s/conslbl-%s"
+	RnL3extConsLbl        = "conslbl-%s"
+	ParentDnL3ExtConsLbl  = "uni/tn-%s/out-%s"
+	L3extConsLblClassName = "l3extConsLbl"
+)
+
+type L3ExtConsLbl struct {
+	BaseAttributes
+	L3ExtConsLblAttributes
+}
+
+type L3ExtConsLblAttributes struct {
+	Name string `json:",omitempty"`
+
+	Annotation string `json:",omitempty"`
+
+	NameAlias string `json:",omitempty"`
+
+	Owner string `json:",omitempty"`
+
+	Tag string `json:",omitempty"`
+}
```

Comment:
All of those are fixed now. I thought it was some kind of convention, because I see in other resources' code that those spaces are quite common. Thanks for pointing that out.

Reply:
yeah it is not consistent everywhere in older code, but reads bit easier I think so easy change to do make for new code